tests/BUILD

```diff
@@ -96,8 +96,10 @@ cc_test(
 cc_library(
     name = "test_define",
     hdrs = ["test_define.h"],
+    copts = ["-DLOGGER_ENABLE"],
```

Comment:
I wonder the conditional build for flag `LOGGER_ENABLE`. @HowJMay, can you comment this for consistency?

Reply:
The conditional build for flag `LOGGER_ENABLE` is used to enable logger when compiling the sources. Otherwise all logger functions will be excluded from compilation. But I found that dependencies in these targets (`driver`, `test_pow` and `test_serializer`) had enabled `LOGGER_ENABLE`, so it's not necessary to modify `tests/BUILD`.